docs/src/modules/components/AppFrame.js

```diff
@@ -108,7 +108,7 @@ const styles = theme => ({
       duration: theme.transitions.duration.leavingScreen,
     }),
     left: theme.spacing(2),
-    top: theme.spacing(-6),
+    top: theme.spacing(-10),
```

Comment:
Move the banner higher so we don't see it when we scroll
![Capture_d’écran_2019-04-29_à_10 37 31](https://user-images.githubusercontent.com/3165635/56929494-0c1aab80-6ada-11e9-9fdc-2ef416ad8dcf.png)